.eslintrc.json

```diff
@@ -19,5 +19,16 @@
         "react"
     ],
     "rules": {
+
+    },
+    "settings": {
```

Comment:
This is to get rid of the red-squiggly lines under the .jsx imports, correct? I'm trusting you've tested it.

Reply:
Yes, that's right. The configuration I added before stopped the linter, but this new one keeps the linter working and gets rid of the 'error' message for jsx imports.